pageserve/test.py

```diff
@@ -16,11 +16,20 @@
 import unittest
 import app
 
+
 class TestServe(unittest.TestCase):
-    def test_get_auth(self):
-        self.assertTrue(app.get_auth("carolyn"))
-        self.assertFalse(app.get_auth("Voldemort"))
-        
-        
+    def test_auth_json(self):
+        response = app.get_auth_json('example_user')
+        # edit_access should be either True or False
+        valid_response = response['edit_access'] or not response['edit_access']
```

Comment:
Eventually, you should create two tests, one that tests yes-auth, and one that tests no-auth case.